src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -426,6 +426,12 @@ bool InputStateMachineEngine::ActionCsiDispatch(const VTID id, const VTParameter
     case CsiActionCodes::DTTERM_WindowManipulation:
         success = _pDispatch->WindowManipulation(parameters.at(0), parameters.at(1), parameters.at(2));
         break;
+    case CsiActionCodes::FocusIn:
+        success = _pDispatch->FocusChanged(true);

```

Comment:
I'm surprised this works in VT Input Passthrough mode!

Reply:
.... me too....

Reply:
(does it??)

Reply:
Oh no this doesn't work for WSL at all. Poop.

Reply:
(fixed here or in #12899. It's all one stack).